libs/platform/src/lib/components/form/radio-group/radio/radio.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormModule as FdFormModule, RadioModule } from '@fundamental-ngx/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RadioButtonComponent } from './radio.component';
+
+@NgModule({
+    imports: [CommonModule, FdFormModule, RadioModule, FormsModule, ReactiveFormsModule],
+    exports: [RadioButtonComponent],
+    declarations: [RadioButtonComponent]
+})
```

Comment:
We said that we dont need this module, radio will be part of RadioGroup. Unless it make sense to use this on its own. What do you think?